connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -253,10 +253,20 @@ public void tick() {
                 boolean remains = configState.connectors().contains(connectorName);
                 log.info("Handling connector-only config update by {} connector {}",
                         remains ? "restarting" : "stopping", connectorName);
-                worker.stopConnector(connectorName);
+                try {
+                    worker.stopConnector(connectorName);
+                } catch (ConnectException e) {
+                    log.error("failed to stop connector " + connectorName, e);
```

Comment:
If this fails, is it really safe to continue? We're still missing status tracking and a corresponding API, but can we just continue and restart the connector if stopping the old one failed?

Nit: Capitalization on the error message.